include/measurement_kit/json/noexceptions.hpp

```diff
@@ -0,0 +1,29 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef MEASUREMENT_KIT_COMMON_NOEXCEPTIONS_HPP
+#define MEASUREMENT_KIT_COMMON_NOEXCEPTIONS_HPP
+
+#include <measurement_kit/common/error_or.hpp>
+
+namespace mk {
```

Comment:
This probably should live in the `mk::json` namespace. Also, it would be nice to make `nlohmann::json` available as `mk::nlohmann::json` through a wrapper header (i.e. `<measurement_kit/json/json.hpp>` that includes the real file stored as `<measurement_kit/json/nlohmann/json.hpp>`. Also, if you have not already done that, I would make sure that the code for updating the `nlohmann/json.hpp` header is update to download the file in the correct location.